lte/gateway/c/oai/tasks/mme_app/mme_app_state_converter.cpp

```diff
@@ -284,13 +267,8 @@ void MmeNasStateConverter::proto_to_ue_mm_context(
   const UeContext* ue_context_proto,
   ue_mm_context_t* state_ue_mm_context)
 {
-  state_ue_mm_context->imsi = ue_context_proto->imsi();
-  state_ue_mm_context->imsi_len = ue_context_proto->imsi_len();
-  state_ue_mm_context->member_present_mask =
-    ue_context_proto->member_present_mask();
-  strcpy(
-    (char*) state_ue_mm_context->imeisv.u.value,
-    ue_context_proto->imeisv().c_str());
+  state_ue_mm_context->emm_context._imsi64 = ue_context_proto->imsi();
```

Comment:
The state_ue_mm_context->emm_context struct is updated via the emm_context_to_proto function call. So lines 270 and 271 are not needed.

Reply:
Updated in PR 766